cql3/Cql.g

```diff
@@ -849,7 +849,8 @@ indexIdent returns [::shared_ptr<index_target::raw> id]
     @init {
         std::vector<::shared_ptr<cql3::column_identifier::raw>> columns;
     }
-    : c=cident                   { $id = index_target::raw::values_of(c); }
```

Comment:
This first patch doesn't fix #8745, the entire series does. The first patch alone isn't sufficient.

Reply:
Issue #8745 was not about fully implementing VALUES() but just adding its syntax. Adding its syntax means we get the right error message for values(v) when v is a **frozen** set (the error should be that v is frozen, not a syntax error). I think it is fine that we mark 8745 fixed as soon as the syntax is added - even if values(v) for a non-frozen set still doesn't work (just like keys(v), etc., doesn't work).